File: study08/backend/springframework5/company/src/main/java/com/company/domain/Member.java
package com.company.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Member {
    private String id;
    private String pw;
    private String name;
    private String birth;
    private String email;
    private String addr1;
    private String addr2;
    private String postcode;
    private String regdate;
}

File: study08/backend/next/company/models/Member.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database'); // 데이터베이스 설정 파일을 import

const Member = sequelize.define('Member', {
  id: {
    type: DataTypes.INTEGER,
    autoIncrement: true,
    primaryKey: true
  },
  name: {
    type: DataTypes.STRING,
    allowNull: false
  },
  email: {
    type: DataTypes.STRING,
    allowNull: false
  },
    birth: { type: DataTypes.DATE, allowNull: false },
    email: { type: DataTypes.STRING, allowNull: false },
    tel: { type: DataTypes.STRING },
    addr1: { type: DataTypes.STRING },
    addr2: { type: DataTypes.STRING },
    postcode: { type: DataTypes.STRING },
    regdate: { type: DataTypes.DATE, defaultValue: DataTypes.NOW }
}, {
    tableName: 'member',
    timestamps: false
});

module.exports = Member;